modules/ROOT/pages/dataweave-extension-plugin.adoc

```diff
@@ -0,0 +1,341 @@
+= Installing and Using the DataWeave Extension Plugin
```

Comment:
Is "DataWeave Extension Plugin" the name. Extension = Plugin 
VS Code uses "extension". 

Maybe "DataWeave Extension to Visual Studio Code"